services/worker/pyproject.toml

```diff
@@ -9,48 +9,25 @@ license = "Apache-2.0"
 python = "3.9.18"
 aiohttp = "^3.9.1"
 aiolimiter = "^1.0.0"
-bs4 = "^0.0.1"
 conllu = "^4.5.2"
 duckdb = "^0.10.0"
 environs = "^9.5.0"
-gdown = "^4.6.3"
-kenlm = { git = "https://github.com/kpu/kenlm", branch = "master" }
-kss = "^2.6.0"
+faiss-cpu = "^1.8.0"
+h5py = "^3.10.0"
 libcommon = {path = "../../libs/libcommon", develop = true}
-lm-dataformat = "^0.0.20"
 lxml = "^4.9.2"
 mirakuru = "^2.4.2"
-nlp = "^0.4.0"
```

Comment:
how did this end up here ? It's the old `datasets` library xD

Reply:
yes... I added it at some point (surely at the very start of the project), because one of the scripts imported it